variantlib/models/variant.py

```diff
@@ -202,8 +223,10 @@ def hexdigest(self) -> str:
         #     a :: b :: c
         #     xd :: e :: f
         # would serialize to the same hash.
-        for vprop in self.properties:
-            hash_object.update(f"{vprop.to_str()}\n".encode())
+
+        hash_object = hashlib.sha256(
+            "\n".join([vprop.to_str() for vprop in self.properties]).encode()
+        )
```

Comment:
Let's not change this. "Newline-separated string" is a weird concept, and having (n-1) properties with a newline, and nth property without is inconsisent.